Topic 1: Maven (new version)

hherminioandres opened the pull request:
-Maven project created.
-The POM file was customized with dependencies: log4j, junit.
-The simple test was created and executed.

hherminioandres committed:
Topic 1: Maven (new version)

-Maven project created.
-The POM file was customized with dependencies: log4j, junit.
-The simple test was created and executed.

carlos-lopez-glb commented on target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml:
```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
The target folder shouldn't be pushed as part of the project in git

hherminioandres replied:
Unnecessary folder was removed from this project.

carlos-lopez-glb commented on pom.xml:
```diff
@@ -0,0 +1,32 @@
+    	<scope>test</scope>
+    </dependency>
+    
+	<dependency>
```
Please apply formatting to the code

hherminioandres replied:
Formatting made in pom.xml file.

carlos-lopez-glb commented on src/test/java/com/Maven/java_bootcamp/AppTest.java:
```diff
@@ -0,0 +1,23 @@
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+public class AppTest 
```
is missing the last part of the exercise.

> Then skip the unit test by property or by adding the skipping test configuration to your pom.xml file.

hherminioandres replied:
Skipping unit test configuration was added to pom.xml file.

hherminioandres committed:
Topic 1 correction

-Skipping unit test configuration was added to pom.xml file.
-Format aplicated to the code.
-The unnecessary project files was deleted.
